backstop/scenarios/menu.js

```diff
@@ -1,7 +1,26 @@
-const { scenario } = require('../scenarioHelper');
+const { scenario, hover, focus } = require('../scenarioHelper');
 
 module.exports = [
-  scenario('basic', {
+  // Menu types
+  scenario('Type Default', {
+    selectors: ['#demo-default'],
+  }),
+  scenario('Type Header', {
```

Comment:
Do we really want to have tests for context? Because I really like this idea of having context but maybe don't flood our tests.

Reply:
I've removed the tests for selects & dropdown buttons because the menu itself was the same as the default test.  I left the header test in because it is styled slightly different to get the image / icon to align properly.